x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_header/cards/common/link_card.tsx

```diff
@@ -50,16 +63,26 @@ export const LinkCard: React.FC<LinkCardProps> = React.memo(
             size={64}
           />
         }
-        title={<span className="headerCardTitle">{title}</span>}
+        title={
+          <span id={panelTitleId} className="headerCardTitle">
+            {title}
+          </span>
+        }
         description={<EuiText size="xs">{description}</EuiText>}
       >
         <EuiSpacer size="s" />
-        <EuiText size="xs" className="headerCardLink">
-          {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
-          <EuiLink data-test-subj="headerCardLink" href={href} onClick={onClick} target={target}>
-            {linkText}
-          </EuiLink>
-        </EuiText>
+        <EuiFlexGroup alignItems="center" gutterSize="xs" responsive={false}>
+          <EuiFlexItem grow={false}>
+            <EuiText size="xs" className="headerCardLink" component="span">
+              <span data-test-subj="headerCardLink">{linkText}</span>
+            </EuiText>
+          </EuiFlexItem>
+          {!isVideoCard && (
```

Comment:
consider checking `target` instead:
```suggestion
          {target === '_black' && (
```